ci/unit.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.17.6
+    tag: latest
```

Comment:
shouldn't CI match prod?

Reply:
I discussed this with Rhys (@redhug1), and it was decided some time ago that it would be better if we used the latest version, so that we test things according to this. 

I think the proper solution would be to use our own registry, and decide what image to use from there, but it seems it's not going to happen for now.